build.rs

```diff
@@ -112,14 +112,22 @@ fn build_nasm_files() {
     let mut config_include_arg = String::from("-I");
     config_include_arg.push_str(&out_dir);
     config_include_arg.push('/');
-    nasm_rs::compile_library_args(
-      "rav1easm",
-      asm_files,
-      &[&config_include_arg, "-Isrc/"],
-    )
-    .expect(
+    let mut nasm = nasm_rs::Build::new();
+    for file in asm_files {
+        nasm.file(file);
```

Comment:
```suggestion
      nasm.file(file);
```

Should make rustfmt happy